Packs/ThreatGrid/Playbooks/playbook-Detonate_File_-_ThreatGrid.yml

```diff
@@ -484,101 +361,21 @@ inputs:
   value:
     complex:
       root: File
-  required: false
+  required: true
   description: File object of the file to detonate.
-  playbookInputQuery:
-- key: FileName
-  value:
-    simple: file-detonated-via-demisto
-  required: false
-  description: Name of the file to detonate.
-  playbookInputQuery:
-- key: VM
-  value: {}
-  required: false
-  description: The VM to use (string)
-  playbookInputQuery:
-- key: playbook
-  value:
-    simple: default
-  required: false
-  description: Name of the Threat Grid playbook to apply to this sample run
-  playbookInputQuery:
-- key: Private
-  value:
-    simple: "True"
-  required: false
-  description: If the value is set to 'False', the sample will not be private. Any value other than 'False' will set the sample to private.
-  playbookInputQuery:
-- key: Source
-  value: {}
-  required: false
-  description: a string used for identifying the source of the detonation (user defined)
-  playbookInputQuery:
-- key: Tags
-  value: {}
-  required: false
-  description: A comma-separated list of tags applied to this sample.
-  playbookInputQuery:
-- key: Interval
-  value:
-    simple: "1"
-  required: false
-  description: Polling frequency - how often the polling command should run (minutes)
-  playbookInputQuery:
-- key: Timeout
-  value:
-    simple: "15"
-  required: false
-  description: How much time to wait before a timeout occurs (minutes)
-  playbookInputQuery:
+  playbookInputQuery: null
 outputs:
-- contextPath: File.Malicious
-  description: The File malicious description
-  type: unknown
-- contextPath: File.Malicious.Vendor
-  description: For malicious files, the vendor that made the decision
-  type: string
 - contextPath: File.Type
-  description: File type e.g. "PE"
+  description: File Type
   type: string
 - contextPath: File.Size
-  description: File size
+  description: File Size
   type: number
-- contextPath: File.MD5
-  description: MD5 hash of the file
-  type: string
-- contextPath: File.Name
-  description: Filename
+- contextPath: File.Extension
+  description: File Extension
   type: string
-- contextPath: File.SHA1
-  description: SHA1 hash of the file
-  type: string
-- contextPath: File
-  description: The File object
-  type: unknown
-- contextPath: File.SHA256
-  description: SHA256 hash of the file
-  type: string
-- contextPath: DBotScore
-  description: The DBotScore object
-  type: unknown
-- contextPath: DBotScore.Indicator
-  description: The indicator we tested
-  type: string
-- contextPath: DBotScore.Type
-  description: The type of the indicator
-  type: string
-- contextPath: DBotScore.Vendor
-  description: Vendor used to calculate the score
-  type: string
-- contextPath: DBotScore.Score
-  description: The actual score
-  type: number
-- contextPath: ThreatGrid.Sample.State
-  description: The sample state.
-- contextPath: ThreatGrid.Sample.ID
-  description: The sample ID.
```

Comment:
why are all these removed? Is this by design? @erezh31 @TalGumi

Reply:
We updated this playbook as Danil asked on 1/1/23.

Reply:
@TalGumi Playbooks should act the same as integrations - We deprecate old ones and release a V2

Reply:
OK. I will restore the old playbooks.